Dump documentation (#468)
meilisearch/documentation: guides/advanced_guides/configuration.md

- [Payload Limit Size](/guides/advanced_guides/configuration.md#payload-limit-size)
- [Max MDB Size](/guides/advanced_guides/configuration.md#max-mdb-size)
- [Max UDB Size](/guides/advanced_guides/configuration.md#max-udb-size)

> - [Backup](/guides/advanced_guides/configuration.md#backup-path)

Reviewer: Since this is way more important for the user than `Max udb size` and `Max mdb size` I would suggest you move it above Max MDB size

[To know more about storage in MeiliSearch look at this guide](/resources/about_storage.md)

**Default value**: `107374182400` (100 GiB)

### Backup path

**Environment variable**: `MEILI_BACKUP_PATH`

> **CLI option**: `--backup-path`

Reviewer 2: ```suggestion
**CLI option**: `--backup-folder`
```

> Path where backup will be created if [backup route](/references/backup.md#trigger-backup) is called.

Reviewer: ```suggestion
Path where backup's will be created if the [backup route](/references/backup.md#trigger-backup) is called.
```

Reviewer 2: ```suggestion
Path where backups will be created if the [backup route](/references/backup.md#trigger-backup) is called.
```

[To know more about storage in MeiliSearch look at this guide](/resources/about_storage.md)

**Default value**: `107374182400` (100 GiB)

### Dumps folder

**Environment variable**: `MEILI_DUMPS_FOLDER`

> **CLI option**: `--dumps-folder`

Author: dumps-path is not clear when you have the path of the folder containing all dumps (`/dumps`) and the path of one dump (`/dumps/202009-120423045.tar.gz`)
In can propose `--dumps-folder-path` or `--dumps-dir-path` which is longer and need to modify v0.15.0 prerelease

Path of the folder where dumps will be created if the [dump route](/references/dump.md#trigger-dump) is called.

**Default value**: `dumps/`

### Import dump

**Environment variable**: `MEILI_IMPORT_DUMP`
**CLI option**: `--import-dump`

> Import a dump from the specified path, must be a `.tar.gz` file.

Author: dump is imported/indexed before meilisearch-http run

> Import a dump from the specified path, must be a `.tar.gz` file.

Reviewer: Seems like a good suggestion ! Could you change the typo here : 
`the process will take som etime`
Its `the process will take some time` :) 
For some reason I cannot comment beneath

> Import a dump from the specified path, must be a `.tar.gz` file.

Reviewer 3: Done @bidoubiwa !

### Dump batch size

**Environment variable**: `MEILI_DUMP_BATCH_SIZE`
**CLI option**: `--dump-batch-size`

> The batch size used in the importation process, the bigger it is the faster the dump is created.

Author: Document indexation during dump importation are batched:
this mean that if you have `--dump-batch-size 1000`  and the dump contains 2600 documents, the engine will
1) index documents 0 -> 999 (1000 docs)
2) index documents 1000 -> 1999 (1000 docs)
3) index documents 2000 -> 2599 (600 docs)

instead of indexing all of 3600 docs in same time.

more you index documents in the same batch less time it takes but more memory is used.

> The batch size used in the importation process, the bigger it is the faster the dump is created.

Reviewer: I'm sorry I was not explicit. Could you add that to the documentation?

```suggestion
The batch size used in the importation process. The more you index documents in the same batch the less time it takes but more memory is used.

**Example**
Imagine you have `--dump-batch-size 1000` and the dump contains 2600 documents. Instead of indexing all of 3600 docs in one go, the engine will :

1. Index documents 0 -> 999 (1000 docs)
2. Index documents 1000 -> 1999 (1000 docs)
3. Index documents 2000 -> 2599 (600 docs)
```

**CLI option**: `--ignore-snapshot-if-db-exists`

The engine skips snapshot importation if a database already exists. No error is thrown in this case.

### Dumps folder

**Environment variable**: `MEILI_DUMPS_FOLDER`
**CLI option**: `--dumps-folder`

> Path of the folder where dumps will be created if the [dump route](/references/dump.md#trigger-dump) is called.

Reviewer: ```suggestion
Path of the folder where dumps will be created if the [dump route](/references/dump.md#trigger-a-dump-creation) is called.
```

I think this should fix the `check-link` test
